### DISCUSSION TITLE
Ability to add helper functions

### DISCUSSION THREAD

**Discussion by Wulfheart** [spatie/laravel-package-tools#9] [Ideas]
Proposed syntax:

````
$package->helpers([
'path'
]);
````

**Comment by abr4xas** [comment]
Hey @Wulfheart I think that you can add you custom helpers in your packages and add them to your composer.json package.

Maybe something like this:

```json
{
  "autoload": {
    "files": [
      "path/to/Helpers.php"
    ]
  }
}
```

Hope this makes sense to you 😃

**Reply by Wulfheart** [reply]
This is even nicer!
